### ISSUE TITLE
Default Branch

### ISSUE THREAD

**Issue by kunicmarko20** [FriendsOfSymfony/FOSCKEditorBundle#13]
My suggestion is that we create `1.x` from `master` and continue our work there. So `1.x` would be stable and `master` unstable. We merge `1.x` into master and from master we release next majors.

**Comment by kunicmarko20** [comment]
We can do this at the end, when the bundle is ready to be released we create new branch from master and use it as a stable for that version

**Comment by stof** [comment]
IMO, we don't need to maintain 2 branches yet. I don't think we need to start working on the second major version already. Maintaining a single branch is much less work (and bundles under the FOS umbrella must take care to BC anyway, as our users expect it from us due to the way we worked until now).

Branching 1.x should be done only when master becomes the dev version of 2.x rather than of a next 1.y version.

**Comment by kunicmarko20** [comment]
Makes sense, we should leave only master then until we start thinking about 2.x

**Comment by stof** [comment]
yeah, exactly
